### ISSUE TITLE
Noting widget relationships

### ISSUE THREAD

**Issue by waywardmonkeys** [linebender/xilem#1003]
When doing accessibility in the browser, one often use aria attributes for ["described by"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-describedby) or[ "labelled by"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-labelledby).

Right now, Masonry doesn't support this, but it would be useful to have.

This is especially true for something like a spinner, which has no label in Masonry, but would find such a thing useful for accessibility support.

**Comment by waywardmonkeys** [comment]
The question is that if we're doing this sort of thing for AccessKit, we need a capability similar to what the aria attributes provide in HTML.
